test/studies/lcals/LCALSMain.graph

```diff
@@ -0,0 +1,14 @@
+perfkeys: ENERGY_CALC, VOL3D_CALC, DEL_DOT_VEC_2D, COUPLE, INIT3, MULADDSUB, IF_QUAD, TRAP_INT, HYDRO_1D, INNER_PROD, FIND_FIRST_MIN
```

Comment:
I think this is good for now, but long term as all the kernels and variations of the kernels are added we'll probably want to split this out into a graph per kernel.